frontend/src/bundles/common/components/checkbox/checkbox.tsx

```diff
@@ -12,7 +12,7 @@ type Properties = {
 
 const Checkbox: React.FC<Properties> = ({
     label,
-    checked = false,
+    checked,
```

Comment:
Why it's been changed?

Reply:
I changed defaultChecked to checked, and `checked = false` would make checkbox to be only false if checked prop is not provided